README.md

```diff
@@ -0,0 +1,8 @@
+# KubeEdge Examples
+
+This repository contains some sample applications and demos which are created to use along with [KubeEdge](https://github.com/kubeedge/kubeedge) platform
```

Comment:
This repository contains some sample applications and demos to illustrate possible use cases of KubeEdge (link)... lets not have platform work at the end